### ISSUE TITLE
[Bug] In the daylight build, the srcollar remains unscrollable

### ISSUE THREAD

**Issue by Plexprofile** [Floorp-Projects/Floorp#757]
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

https://github.com/Floorp-Projects/Floorp/assets/139643082/064a7ec4-a5e7-4ea9-811b-3c45eea5a1b6

### To Reproduce

move the scrollbar

### Expected behavior

The issue of the scroll bar being unscrollable persists in the new update 

### OS Info

Win10 21h2

### Floorp Version

Version 11.8.2 (64-bit)

**Comment by surapunoyousei** [comment]
I have fine. Try this on new profile

**Comment by Plexprofile** [comment]
The issue persists with a new profile. I've even tried installing floorp daylight with a new administrator account, but that still fails to fix the problem

**Comment by Plexprofile** [comment]
By the way floorp daylight by default start with a new user profile with a first-time-install, so the video I recorded was recorded was recorded in a new profile

**Comment by menndouyukkuri** [comment]
Windows 11 23H2で安定版の11.8.2、Daylightの11.8.2の両方で発生することを確認しました。
https://github.com/Floorp-Projects/Floorp/assets/61068665/d8e3e054-2a59-4734-a226-ddf77e897c83

Zorin OS 17 Core（Linux）では発生しませんでした。

**Comment by surapunoyousei** [comment]
条件わかんないし再現できない

**Comment by Not-A-Coder2023** [comment]
Another weird issue is that the _Firefox offline translations_ tend to not show up under the _Translate Web Pages_ addon box in **about:preferences**. Has this happened to anybody else?

**Comment by BombasticBooger** [comment]
I also have this issue, Windows 11 23h2, Floorp 11.9.0

**Comment by brokedarius** [comment]
I have this exact issue as well on win 10 22h2 and ver. 11.9.0

**Comment by shanoaice** [comment]
This issue have just appeared to me after an Floorp (and Windows Update). Little idea of what causes it. Win 11 23H2 22631.3593, Floorp 11.13.1.

**Comment by surapunoyousei** [comment]
Yeah. I reproduced this issue. I will send a patch for fix this issue

**Comment by shanoaice** [comment]
Thanks, though opening a new profile appears to fix the issue. Should I just reset the browser as a temporary workaround?

**Comment by surapunoyousei** [comment]
Hey everyone. It issue maybe fixed. Please check latest version of Floorp.

**Comment by Dobfek** [comment]
Hi! I have the exact same issue with a fresh installation of version 11.14.1(64 bits). The scrollbar is not working, only works with the arrows. It seems like the scrollbar has titlebar properties. If you double click it, the window goes full size-previous size vice-versa. If you right click it, it has the titlebar right click menu, and you can move the whole window with it instead of scrolling like with a titlebar grabbing.

**Comment by surapunoyousei** [comment]
What is "titlebar property" means?

**Comment by Dobfek** [comment]
> What is "titlebar property" means?

I meant that it completely works as a titlebar instead of working as a scrollbar. Quite strange.

**Comment by Smyl3** [comment]
It's still an issue for me on version 11.18.1 64bit Win10.
I just wanted to make an issue ticket for this but I found it here already.
Clicking on scrollbar is moving window, not acting as a scrollbar.
